packages/manager/apps/dedicated/client/app/ip/dashboard/reverse/index.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-ui-angular';
+
+import add from './add';
+import deleteModule from './delete';
+import update from './update';
+
+import component from './reverse.component';
+import routing from './reverse.routing';
+import service from './reverse.service';
+
+const moduleName = 'ovhManagerDedicatedIpDashboardReverse';
+
+angular
+  .module(moduleName, [
+    add,
+    deleteModule,
+    'oui',
+    'pascalprecht.translate',
+    'ui.router',
+    update,
+  ])
+  .component('ipDashboardReverse', component)
```

Comment:
I see a lot of `$scope` used in the controller. Aren't there any issue with the use of component to inherit parent scope ?

Reply:
No, I've checked every `$scope` to make sure they use only data from the current view, and if they don't, I've created a new component (you can see an example here: https://github.com/ovh/manager/pull/2766/commits/cf1d7a71320a4d2bbca1ef4b0abebb54e4f4585a)